packages/message-relayer/src/service.ts

```diff
@@ -78,7 +78,7 @@ export class MessageRelayerService extends BaseService<MessageRelayerOptions> {
   }
 
   protected async _init(): Promise<void> {
-    this.logger.info('Initializing message relayer', { options: this.options })
```

Comment:
Hmmm could we make a copy of the object instead and simply delete the secret options? I feel like this might come in handy one day.